File: ClientApp/src/app/Shared/Entity/WorkFlows/workflow-log.ts
import { WorkflowLogHistory } from './workflow-log-history';

export class WorkflowLog {

    public  id : number;
    public  rowId : number; 
    public  workFlowFor : number;
    public  masterWorkFlowId : number;
    public  workflowStatus : number;
    public  pendingWorkflowCount : number;
    public  workflowMessage: string;
    public  createdTime: Date;
    public  data: any;
    public  logHistories : WorkflowLogHistory[];

    constructor()
    {
        this.logHistories = [];
    }

}

File: Berger.Data/MsfaEntity/WorkFlows/WorkflowLog.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using Berger.Data.Common;

namespace Berger.Data.MsfaEntity.WorkFlows
{

    public class WorkflowLog : AuditableEntity<int>
    {
        public int RowId { get; set; }
        public int WorkFlowFor { get; set; }
        public int MasterWorkFlowId { get; set; }
        public string TableName { get; set; }
        public int OrgRoleId { get; set; }
        public int WorkflowStatus { get; set; }
        [ForeignKey("MasterWorkFlowId")]
        public WorkFlow WorkFlow { get; set; }
        public List<WorkflowLogHistory> Histories { get; set; }


    }
}
